storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/design/GetOperationDesignTest.java

```diff
@@ -40,4 +40,17 @@ public void getListOfObjectsBlocking() {
                 .prepare()
                 .executeAsBlocking();
     }
+
+    @Test
+    public void geObjectBlocking() {
```

Comment:
`getObjectBlocking()` (missing `t`)

BTW, did you find our design tests useful? Just curious :)